docker-compose.yml

```diff
@@ -18,6 +18,39 @@ services:
       - phabricator.test
       - bmo.test
 
+  lando-ui.test:
+    image: mozilla/landoui
```

Comment:
I feel like we have some inconsistent image naming. In tasks.py of mozilla-services/lando-ui, we have the image name as mozilla/landoui. So we should change that or change this to:

image: mozilla/lando-ui